src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -141,6 +145,12 @@ private async ValueTask InitializeAsync(PackageManagerModel model, INuGetUILogge
             _settingsKey = await GetSettingsKeyAsync(CancellationToken.None);
             UserSettings settings = LoadSettings();
             InitializeFilterList(settings);
+            var isReadmeTabEnabled = await _nugetFeatureFlagService.IsFeatureEnabledAsync(NuGetFeatureFlagConstants.RenderReadmeInPMUI);
+
+            _nugetPackageFileService?.Dispose();
+            _nugetPackageFileService = await _serviceBroker.GetProxyAsync<INuGetPackageFileService>(NuGetServices.PackageFileService, CancellationToken.None);
+            _packageDetail._packageDetailsTabControl.PackageDetailsTabViewModel.Initialize(_detailModel, _nugetPackageFileService, _topPanel.Filter, isReadmeTabEnabled);
+            InitializeSelectedPackageMetadataTab(settings, isReadmeTabEnabled);
```

Comment:
It'd probably be better to orchestrate the details pane's tabs in the DetailControlModel rather than in the View. This would also reduce how far and wide the Tab control reaches across these Views and ViewModels (eg, here in the PackageManagerControl).

Reply:
Would this mean adding a reference to the PackageDetailsTabViewModel inside of the DetailControlModel? Then changes are all handled in that VM?